Tests/Tests/DatastoreCRUD.m

```diff
@@ -83,6 +83,41 @@ -(NSArray*)generateDocuments:(int)count
 
 #pragma mark - CREATE tests
 
+
+-(void) testDocumentWithInfinityValue
+{
+    NSError * error;
+    CDTMutableDocumentRevision * revision = [CDTMutableDocumentRevision revision];
+    revision.body = @{@"infinity":@(INFINITY)};
+    
+    CDTDocumentRevision * rev = [self.datastore createDocumentFromRevision:revision
+                                                                     error:&error];
+    
+    XCTAssertNil(rev,@"revision is not nil");
+    XCTAssertNotNil(error,@"Error should be set");
+    XCTAssertEqual(400,
```

Comment:
493 isn't a http status so it is mapped to 400. https://github.com/cloudant/CDTDatastore/blob/master/Classes/common/touchdb/TDStatus.m#L42